### PR TITLE
Fix codepen visuals

### PR DESCRIPTION
**Pull request by esthermmoturi**
This fixes the issue described here https://github.com/medic/cht-docs/issues/1560

### CODE REVIEW

**Comment by esthermmoturi** [comment]
Hey @abbyad , feel free to provide feedback as well.This is a snippet of what it looks like.
image 
image

**Review by n-orlowski — APPROVED** [review]
Thanks for fixing!

File: content/en/design/components/_index.md [hunk 1]
```diff
@@ -74,44 +74,44 @@ H4 is the default type size, and should be used for all normal body text through
...
 
 ## Radios
 Use radio buttons to select a single item within a form. When necessary, use a descriptive error state.
-        {{% codepen RwrLqqZ %}}
+        {{< codepen RwrLqqZ >}}
 
 ## Inputs
 Use an input field as a basic text control to enable the user to type a small amount of text. The input size should reflect the amount of text a user is expected to enter. When necessary, use a descriptive error state.
-        {{% codepen KKVXrYE %}}
+        {< codepen KKVXrYE >}}
```
Comment (abbyad) [inline content/en/design/components/_index.md:93]: @esthermmoturi, this correction accidentally removed one of the brackets.

Comment (esthermmoturi) [reply content/en/design/components/_index.md:93]: Thanks @abbyad for catching this. Fixed by Nicole on https://github.com/medic/cht-docs/pull/1585

**Pull request merged**